### ISSUE TITLE
Requested task are visible only per session

### ISSUE THREAD

**Issue by jamuszyn** [golemfactory/golem-monitor-frontend#6]

**Comment by tworec** [comment]
this is because only provider stats are persisted at golem nodes

https://github.com/golemfactory/golem/blob/7c0f09601ed97e4ae3b54ff460190468063622f7/golem/core/statskeeper.py#L55-L62

**Comment by jamuszyn** [comment]
But it seems that it is like that without any particular reason, so should be changed at some point.

**Comment by tworec** [comment]
yup, thats for sure. But to achieve this one need to fill an issue within the [golem](https://github.com/golemfactory/golem/) repo.
